pychunkedgraph/graph_analysis/analysis.py

```diff
@@ -0,0 +1,266 @@
+import fastremap
```

Comment:
I feel this is better placed inside `graph`, maybe `graph.analysis.py` module or `graph.analysis` sub package if you expect to add more stuff overtime

Reply:
also keep analysis code together but move meshing code to `pychunkedgraph.meshing`

Reply:
I agree. I moved it to a new graph.analysis subpackage